### ISSUE TITLE
Pass more options to acorn parser

### ISSUE THREAD

**Issue by dhakehurst** [aMarCruz/rollup-plugin-cleanup#11]
I am trying to 'rollup' the monaco-editor. I get an error with cleanup,

node_modules/monaco-editor/esm/vs/language/typescript/monaco.contribution.js
SyntaxError: Unexpected token (157:31)
    at Parser.pp$4.raise (.../node_modules/acorn/dist/acorn.js:2756:13)

the offending line in monaco.contribution.js is

`return monaco.Promise.wrap(import('./tsMode.js'));`

i.e. use of an import statement.

the documentation for the acorn parser options has an entry

> allowImportExportEverywhere: By default, import and export declarations can only appear at a program's top level. Setting this option to true allows them anywhere where a statement is allowed.

I think I need to be able to set this option, and maybe some others.

**Comment by aMarCruz** [comment]
@dhakehurst thanks for your feedback, I will review this right now.

**Comment by dhakehurst** [comment]
You can pass options to the acorn parser of the main rollup. I just
discovered.

Not sure if you can use this, would best if you can.

**Comment by aMarCruz** [comment]
@dhakehurst yes, it is possible. I will add `acornOptions` for advanced users.

**Comment by aMarCruz** [comment]
@dhakehurst , the option `allowImportExportEverywhere` is not used to support dynamic `import()`. This is an Stage 3 feature that you can support through Acorn plugins.

Just I made a commit supporting a new `acornOptions` option that allows pass additional settings to acorn. You can use the [acorn-dynamic-import](https://github.com/kesne/acorn-dynamic-import) plugin with this option to support `import()`. An [example](https://github.com/aMarCruz/rollup-plugin-cleanup#using-acorn-plugins) is in the README.

Please update the new version in your package.json with the one in the master branch:
```json
  "rollup-plugin-cleanup": "aMarCruz/rollup-plugin-cleanup",
```

...try it out and let me know if it solves your problem. After this I will publish that version in npm.

**Comment by aMarCruz** [comment]
@dhakehurst ,
rollup-plugin-cleanup is already published in npm with (again) 100% coverage.
Ping me if you have troubles with this version and I will fix it in 3.1
Thanks.

**Comment by aMarCruz** [comment]
@dhakehurst , acorn was removed in cleanup 3.1.0
Details in the [Changelog](https://github.com/aMarCruz/rollup-plugin-cleanup/blob/master/CHANGELOG.md)
